standards/coding-in-the-open.md

```diff
@@ -6,9 +6,9 @@ order: 1
 
 ## Measure
 
-* All source code for any used policy and software not used for fraud detection MUST be published and publicly accessible.
-* Any source code that is not currently in use, such as new versions, proposals or older versions SHOULD be published.
-* Contributors MUST never upload sensitive information regarding users, the organization or third parties to the repository, for example: configurations, usernames and passwords, public keys and other real credentials used in the production system.
+* All source code and used policy and software not used for fraud detection MUST be published and publicly accessible.
+* Any source code that is not currently in use, such as new versions, proposals or older versions SHOULD be published. 
+* Contributors MUST never upload sensitive information regarding users, the organization or third parties to the Public Code repository, for example: configurations, usernames and passwords, public keys and other real credentials used in the production system.
```

Comment:
This standard should be applicable to anyone, also to others, so specifying the repository as ours isn't necessary. Apart from that the user should never upload these things to any repository anyways.